api/index.py

```
@@ -309,3 +310,43 @@
+    path      = data.get('path')
+    ignore    = data.get('ignore', [])
+
+    # Validate 'path' parameter
+    if not path:
+        logging.error("'path' is missing from the request.")
+        return jsonify({"status": "'path' is required."}), 400
+
+    proj_name = Path(path).name
+
+    # Initialize the graph with the provided project name
+    g = Graph(proj_name)
```

Ruff (0.8.0)

`Graph` may be undefined, or defined from star imports

(F405)